tests/csm/rest/test_query_deployment.py

```diff
@@ -0,0 +1,111 @@
+# pylint: disable=too-many-lines
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""Tests Query Deployment scenarios using REST API
+"""
+
+import time
+import logging
+
+import pytest
+
+from commons import configmanager, cortxlogging
+from commons.constants import K8S_SCRIPTS_PATH, K8S_PRE_DISK
+from commons.utils import assert_utils
+from libs.csm.csm_interface import csm_api_factory
+from libs.ha.ha_common_libs_k8s import HAK8s
+from libs.prov.prov_k8s_cortx_deploy import ProvDeployK8sCortxLib
+
+
+class TestQueryDeployment():
+    """Query Deployment Testsuites"""
+
+    def __init__(cls):
```

Comment:
This should be setup_class and not __init__

Reply:
changed to setup_class